runtime/jvmti/jvmtiRawMonitor.c

```diff
@@ -259,9 +259,43 @@ jvmtiRawMonitorWait(jvmtiEnv *env,
 			break;
 		case J9THREAD_INTERRUPTED:
 			/* FALLTHROUGH*/
-		case J9THREAD_PRIORITY_INTERRUPTED:
+		case J9THREAD_PRIORITY_INTERRUPTED: {
+#if JAVA_SPEC_VERSION >= 19
+			J9InternalVMFunctions const * const vmFuncs = vm->internalVMFunctions;
+			j9object_t threadObject = NULL;
+			vmFuncs->internalEnterVMFromJNI(currentThread);
+			threadObject = currentThread->threadObject;
```

Comment:
I am wondering whether the assignment with `threadObject` works for the C compiler prior to the initialization for `threadLock` on all supported platform. (e.g. does the compilation pass on Windows?)

Reply:
Yes, Windows won't compile. Fixed it.